toolchain/check/eval.cpp

```diff
@@ -497,6 +510,30 @@ static auto PerformBuiltinCall(Context& context, SemIRLoc loc, SemIR::Call call,
     case SemIR::BuiltinFunctionKind::None:
       CARBON_FATAL() << "Not a builtin function.";
 
+    case SemIR::BuiltinFunctionKind::IntMake32: {
+      if (phase != Phase::Template) {
+        break;
+      }
```

Comment:
I think this check is redundant -- there are no arguments, so there's no phase to check.
```suggestion
```